crates/cadenza-eval/src/generated/snapshots/cadenza_eval__generated__test_data__op_override__op_override.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: crates/cadenza-eval/src/generated/test_data.rs
+expression: "let my_buggy_add = *\nlet + = my_buggy_add\n1 + 2\n"
+---
+EvalResult {
+    values: [
+        Symbol("*"),
+        nil,
+        3,
+    ],
+    diagnostics: [
+        Diagnostic {
+            kind: ArityError {
+                expected: 2,
+                actual: 3,
+            },
```

Comment:
My guess is the parse tree is not what we'd expect, not an issue with the evaluator. We should probably add an AST snapshot as part of the generated test so it's easier to debug why things aren't being evaluated correctly. You can look at the syntax test_data generated test code to see how we'd do it.